src/app/state/movies/movies-state.ts

```diff
@@ -1,8 +1,10 @@
 import { Movie } from './../../interfaces';
 
 export interface MovieState {
+  loading: boolean;
```

Comment:
skąd pomysł żeby to tutaj dawać? tzn to może się przydać ale raczej nie w tej chwili. Flagę loading chcesz ustawiać na razie na każdym filmie osobno, nie na całym state